airflow/models/dag.py

```diff
@@ -3453,12 +3474,11 @@ def get_paused_dag_ids(dag_ids: list[str], session: Session = NEW_SESSION) -> se
         :param session: ORM Session
         :return: Paused Dag_ids
         """
-        paused_dag_ids = (
-            session.query(DagModel.dag_id)
-            .filter(DagModel.is_paused == expression.true())
-            .filter(DagModel.dag_id.in_(dag_ids))
-            .all()
-        )
+        paused_dag_ids = session.execute(
+            select(DagModel.dag_id)
+            .where(DagModel.is_paused == expression.true())
+            .where(DagModel.dag_id.in_(dag_ids))
+        ).all()
```

Comment:
```suggestion
        )
```

I think Mypy would complain here though since `paused_dag_ids` changes type in the next line.